webapp/channels/src/components/user_groups_modal/user_groups_list/user_groups_list.tsx

```diff
@@ -77,97 +109,136 @@ const UserGroupsList = React.forwardRef((props: Props, ref?: React.Ref<HTMLDivEl
         if (groups.length > 1 && groupListItemIndex === 0) {
```

Comment:
To prevent upwards cut off

![CleanShot 2023-08-22 at 13 32 01@2x](https://github.com/mattermost/mattermost/assets/11724372/05edce32-7c09-42b8-80e4-d1d6b00b3e04)


```suggestion
        if (groupListItemIndex === 0) {
```

Reply:
fixed https://github.com/mattermost/mattermost/pull/22597/commits/35aef46f275f1bf8f5612629097e346b2c3d209f